cli/util_internal_test.go

```diff
@@ -41,6 +41,37 @@ func TestDurationDisplay(t *testing.T) {
 	}
 }
 
+func TestExtendedParseDuration(t *testing.T) {
+	t.Parallel()
+	for _, testCase := range []struct {
+		Duration   string
+		Expected   time.Duration
+		ExpectedOk bool
+	}{
+		{"1d", 24 * time.Hour, true},
+		{"1y", 365 * 24 * time.Hour, true},
```

Comment:
nit: should we add test cases for negative durations?

Reply:
I added tests and overall improved negative durations handling to fit with time package.